notebooks/405-paddleOCR-webcam/405-paddleOCR-webcam.ipynb

```diff
@@ -0,0 +1,466 @@
+    "                        key = cv2.waitKey(1)\n",
+    "                        # escape = 27\n",
+    "                        if key == 27:\n",
+    "                            break\n",
+    "                    else:\n",
+    "                        # encode numpy array to jpg\n",
+    "                        draw_img = cv2.cvtColor(draw_img, cv2.COLOR_RGB2BGR)\n",
+    "                        _, encoded_img = cv2.imencode(ext=\".jpg\", img=draw_img,\n",
+    "                                                      params=[cv2.IMWRITE_JPEG_QUALITY, 100])\n",
+    "                        # create IPython image\n",
+    "                        i = display.Image(data=encoded_img)\n",
+    "                        # display the image in this notebook\n",
+    "                        display.clear_output(wait=True)\n",
+    "                        display.display(i)\n",
+    "            \n",
+    "    # ctrl-c\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    # any different error\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "    finally:\n",
+    "        if player is not None:\n",
+    "            # stop capturing\n",
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92f8855f-418a-4bda-8799-0953dda895c5",
+   "metadata": {},
+   "source": [
+    "## Run Live PaddleOCR with OpenVINO"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7642697d-d000-4a10-8e7b-2a519cf9e687",
+   "metadata": {},
+   "source": [
+    "Run using a webcam as the video input. By default, the primary webcam is set with `source=0`. If you have multiple webcams, each one will be assigned a consecutive number starting at 0. Set `flip=True` when using a front-facing camera. Some web browsers, especially Mozilla Firefox, may cause flickering. If you experience flickering, set `use_popup=True`. *Note popup mode may not work if you run this notebook on a remote computer.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc274952-19aa-480d-ba50-a1146a89771b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_paddle_ocr(source=2, flip=False, use_popup=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e9c2356-16e2-4ec9-a41e-ae7d03a25b5e",
+   "metadata": {},
+   "source": [
+    "If you don't have a webcam, you can still run this demo with a video file. Any [format supported by OpenCV](https://docs.opencv.org/4.5.1/dd/d43/tutorial_py_video_display.html) will work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e6c9d077-70df-4a28-a372-7ee5168f6720",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test OCR results on video file\n",
+    "\n",
+    "video_file = \"./data/test_video4.mp4\"\n",
```

Comment:
We should avoid saving data into the repo. Could we use a URL instead of saving the video into the repo?

Reply:
Have uploaded the test video to a URL